src/main/java/org/jabref/gui/preferences/externalfiletypes/EditExternalFileTypeViewModel.java

```diff
@@ -29,6 +46,48 @@ public EditExternalFileTypeViewModel(ExternalFileTypeItemViewModel fileTypeViewM
             customApplicationSelectedProperty.setValue(true);
             selectedApplicationProperty.setValue(fileTypeViewModel.applicationProperty().getValue());
         }
+
+        setupValidation();
+    }
+
+    private void setupValidation() {
+        validator = new CompositeValidator();
+        extensionValidator = new FunctionBasedValidator<>(
+                extensionProperty,
+                StringUtil::isNotBlank,
+                ValidationMessage.error(Localization.lang("Please enter a name for the extension."))
+        );
+
+        nameValidator = new FunctionBasedValidator<>(
+                nameProperty,
+                StringUtil::isNotBlank,
+                ValidationMessage.error(Localization.lang("Please enter a name."))
+        );
+
+        mimeTypeValidator = new FunctionBasedValidator<>(
+                mimeTypeProperty,
+                StringUtil::isNotBlank,
+                ValidationMessage.error(Localization.lang("Please enter a name for the MIME type."))
+        );
+
+        sameExtensionValidator = new FunctionBasedValidator<>(
+                extensionProperty,
+                extension -> {
+                    for (ExternalFileTypeItemViewModel fileTypeItem : fileTypes) {
+                        if (extension.equalsIgnoreCase(fileTypeItem.extensionProperty().get()) && !extension.equalsIgnoreCase(originalExtension)) {
+                            return false;
+                        }
+                    }
+                    return true;
+                },
+                ValidationMessage.error(Localization.lang("There is already an exists extension with the same name."))
```

Comment:
Where do I see these messages? I did not see them here? @Siedlerchr Do we need to change something in the overall code?

Reply:
@koppor 
I following the code from GroupDialogViewModel.java. 
In that class there is a function validationHandler. However even with that function I still cannot see the error message, so I didnt include that.